config/armsve/bli_cntx_init_armsve.c

```diff
@@ -65,20 +65,22 @@ void bli_cntx_init_armsve( cntx_t* cntx )
 	);
 
 	// Set VL-specific packing routines if applicable.
+	// NOTE: SVE-Intrinsic kernels are used without checking __has_include(<arm_sve.h>).
+	//  Such is ensured at configuration stage for config: armsve.
 	if (m_r_d==16)
 	  bli_cntx_set_packm_kers
 	  (
 		3,
 		BLIS_PACKM_10XK_KER, BLIS_DOUBLE, bli_dpackm_armsve512_asm_10xk,
-		BLIS_PACKM_12XK_KER, BLIS_DOUBLE, bli_dpackm_armsve512_asm_12xk,
+		BLIS_PACKM_12XK_KER, BLIS_DOUBLE, bli_dpackm_armsve512_int_12xk,
 		BLIS_PACKM_16XK_KER, BLIS_DOUBLE, bli_dpackm_armsve512_asm_16xk,
 		cntx
 	  );
 	else if (m_r_d==8)
 	  bli_cntx_set_packm_kers
 	  (
 		1,
-		BLIS_PACKM_8XK_KER, BLIS_DOUBLE, bli_dpackm_armsve256_asm_8xk,
+		BLIS_PACKM_8XK_KER, BLIS_DOUBLE, bli_dpackm_armsve256_int_8xk,
```

Comment:
Shouldn't the 10xk kernel also be used here?

Reply:
10xk is for VL=512bits.
Packing kernels are VL-specific, I'm afraid.